docs/config/README.md

```diff
@@ -114,27 +99,14 @@ Let's take a look at a sample config file.
         "region": "eu-west-1"
       },
       "profile": "mira-prod",
-      "withDomain": true,
-      "webAppUrl": "prod.mira-nf.com",
-      "requireManualApproval": true
-    },
-    {
-      "name": "Domain",
-      "env": {
-        "account": "5555555555555",
-        "region": "eu-west-1"
-      },
-      "profile": "mira-prod",
-      "withDomain": true,
-      "webAppUrl": "domain.mira-nf.com",
       "requireManualApproval": true
     }
   ]
 
 }
 ```
 ## Accounts
-We define four accounts in the sample config file above: `Staging`, `Production`, `Default`, `Domain`, each with different account numbers.
+We define four accounts in the sample config file above: `Staging`, `Production`, `Default`, each with different account numbers.
```

Comment:
Staging, Prod, and default would be 3 accounts unless I've missed one?

Reply:
I don't get it. Locally I have `We define three accounts`. Everything is pushed. Can you pull and check what will you see locally?